### PR TITLE
clear translation of anmeldung/models.py

### PR DESCRIPTION
**Pull request by MaxTechCode**

### CODE REVIEW

File: anmeldung/admin.py [hunk 1]
```diff
@@ -92,39 +91,44 @@ class MyCourseAdmin(ImportExportModelAdmin):
...
     search_fields = ['name']
 
+
 class CourseRatingResource(resources.ModelResource):
     class Meta:
         model = CourseRating
 
-        fields = ('event__name','course__name','participant','participant__is_advisor','rating')
-        export_order = ('event__name','course__name','participant','participant__is_advisor','rating')
+        fields = ('event__name', 'course__name', 'participant', 'participant__is_adviser', 'rating')
```
Comment (roflmaostc) [inline anmeldung/admin.py:113]: Should be is_advisor

File: anmeldung/models.py [hunk 2]
```diff
@@ -81,206 +81,223 @@ def pre_save(self, model_instance, add):
...
     # event the participant is connected to
     event = models.ForeignKey(Event,
-                              verbose_name=_("Veranstaltung"),
+                              verbose_name=_("event"),
                               on_delete=models.CASCADE)
 
-    # for advisors. Needs to be set in the admin interface
-    is_advisor = models.BooleanField(_("Betreuer*in"), default=False)
+    # for advisers. Needs to be set in the admin interface
+    is_adviser = models.BooleanField(_("adviser"), default=False)
```
Comment (roflmaostc) [inline anmeldung/models.py:221]: Should be is_advisor

**Comment by roflmaostc** [comment]
Several is_advisor typoes

**Comment by MaxTechCode** [comment]
advisor and adviser is possible
need of change back to old spelling
-> no proper translation tomorrow :-(

**Pull request closed without merging**